### ISSUE TITLE
Error in rendering puml files when executing `npm run testwin`

### ISSUE THREAD

**Issue by le0tan** [MarkBind/markbind#963]
**Tell us about your environment**

Windows 10, node v13.5.0, npm v6.13.4

```
C:\Users\uicfa>java --version
openjdk 13.0.1 2019-10-15
OpenJDK Runtime Environment (build 13.0.1+9)
OpenJDK 64-Bit Server VM (build 13.0.1+9, mixed mode, sharing)
```

* **MarkBind Version:** 2.7.0

**What did you do? Please include the actual source code causing the issue.**

Running `npm run testwin` after cloning the repo, `npm install` and `npm link`.

**What did you expect to happen?**

No error messages.

**What actually happened? Please include the actual, raw output.**

There are error messages when building the test sites. It seems the error only happens when building puml files. However the tests pass in the end.

```bash
C:\Users\uicfa\Documents\Development\markbind>npm run testwin

> markbind-cli@2.7.0 pretestwin C:\Users\uicfa\Documents\Development\markbind
> npm run lint && npm run csslint


> markbind-cli@2.7.0 lint C:\Users\uicfa\Documents\Development\markbind
> eslint .


> markbind-cli@2.7.0 csslint C:\Users\uicfa\Documents\Development\markbind
> stylelint **/*.css !**/*.min.css


> markbind-cli@2.7.0 testwin C:\Users\uicfa\Documents\Development\markbind
> jest && cd test/functional && test.bat

 PASS  test/unit/cliUtil.test.js
 PASS  test/unit/markdown-it-icons.test.js
 PASS  test/unit/Page.test.js
 PASS  test/unit/parser.test.js

 PASS  test/unit/Site.test.js

Test Suites: 5 passed, 5 total
Tests:       49 passed, 49 total
Snapshots:   0 total
Time:        4.762s, estimated 5s
Ran all test suites.

Running test_site tests
  __  __                  _      ____    _               _ 
 |  \/  |   __ _   _ __  | | __ | __ )  (_)  _ __     __| |
 | |\/| |  / _` | | '__| | |/ / |  _ \  | | | '_ \   / _` |
 | |  | | | (_| | | |    |   <  | |_) | | | | | | | | (_| |
 |_|  |_|  \__,_| |_|    |_|\_\ |____/  |_| |_| |_|  \__,_|
                                                           
 v2.7.0
info: Website generation started at 2:45:37 PM
warn: ENOENT: no such file or directory, open 'C:\Users\uicfa\Documents\Development\markbind\test\functional\test_site\sub_site\_markbind\variables.md'
info: Building assets...
info: Assets built
info: Generating pages...
[----------------] 0 / 16 pages builterror: Error processing C:\Users\uicfa\Documents\Development\markbind\test\functional\test_site\diagrams\state.puml
error: Error processing C:\Users\uicfa\Documents\Development\markbind\test\functional\test_site\diagrams\component.puml
error: Error processing C:\Users\uicfa\Documents\Development\markbind\test\functional\test_site\diagrams\activity.puml
error: Error processing C:\Users\uicfa\Documents\Development\markbind\test\functional\test_site\diagrams\sequence.puml
error: Error processing C:\Users\uicfa\Documents\Development\markbind\test\functional\test_site\diagrams\class.puml
error: Error processing C:\Users\uicfa\Documents\Development\markbind\test\functional\test_site\diagrams\usecase.puml
error: Error processing C:\Users\uicfa\Documents\Development\markbind\test\functional\test_site\diagrams\object.puml
[================] 16 / 16 pages built
info: Pages built
info: Website generation complete! Total build time: 1.206s
info: Build success!
info: Site data built

Running test_site_algolia_plugin tests
  __  __                  _      ____    _               _ 
 |  \/  |   __ _   _ __  | | __ | __ )  (_)  _ __     __| |
 | |\/| |  / _` | | '__| | |/ / |  _ \  | | | '_ \   / _` |
 | |  | | | (_| | | |    |   <  | |_) | | | | | | | | (_| |
 |_|  |_|  \__,_| |_|    |_|\_\ |____/  |_| |_| |_|  \__,_|
                                                           
 v2.7.0
info: Website generation started at 2:45:39 PM
info: Building assets...
info: Assets built
info: Generating pages...
[=] 1 / 1 pages built
info: Pages built
info: Website generation complete! Total build time: 0.251s
info: Build success!
info: Site data built

Running test_site_convert tests
  __  __                  _      ____    _               _ 
 |  \/  |   __ _   _ __  | | __ | __ )  (_)  _ __     __| |
 | |\/| |  / _` | | '__| | |/ / |  _ \  | | | '_ \   / _` |
 | |  | | | (_| | | |    |   <  | |_) | | | | | | | | (_| |
 |_|  |_|  \__,_| |_|    |_|\_\ |____/  |_| |_| |_|  \__,_|
                                                           
 v2.7.0
info: Initialization success.
info: Converting to MarkBind website.
info: Copied over the existing _Sidebar.md file to _markbind\layouts\default\navigation.mdCheck https://markbind.org/userGuide/tweakingThePageStructure.html#site-navigation-menus
for information on site navigation menus.
info: The default base URL of your site is set to /
You can change the base URL of your site by editing site.json
Check https://markbind.org/userGuide/siteConfiguration.html for more information.
info: Conversion success.
  __  __                  _      ____    _               _ 
 |  \/  |   __ _   _ __  | | __ | __ )  (_)  _ __     __| |
 | |\/| |  / _` | | '__| | |/ / |  _ \  | | | '_ \   / _` |
 | |  | | | (_| | | |    |   <  | |_) | | | | | | | | (_| |
 |_|  |_|  \__,_| |_|    |_|\_\ |____/  |_| |_| |_|  \__,_|
                                                           
 v2.7.0
info: Website generation started at 2:45:41 PM
info: Building assets...
info: Assets built
info: Generating pages...
[==========] 10 / 10 pages built
info: Pages built
info: Website generation complete! Total build time: 0.396s
info: Build success!
info: Site data built
52 File(s) copied
Running test_site_templates/test_minimal test
  __  __                  _      ____    _               _ 
 |  \/  |   __ _   _ __  | | __ | __ )  (_)  _ __     __| |
 | |\/| |  / _` | | '__| | |/ / |  _ \  | | | '_ \   / _` |
 | |  | | | (_| | | |    |   <  | |_) | | | | | | | | (_| |
 |_|  |_|  \__,_| |_|    |_|\_\ |____/  |_| |_| |_|  \__,_|
                                                           
 v2.7.0
info: Initialization success.
  __  __                  _      ____    _               _ 
 |  \/  |   __ _   _ __  | | __ | __ )  (_)  _ __     __| |
 | |\/| |  / _` | | '__| | |/ / |  _ \  | | | '_ \   / _` |
 | |  | | | (_| | | |    |   <  | |_) | | | | | | | | (_| |
 |_|  |_|  \__,_| |_|    |_|\_\ |____/  |_| |_| |_|  \__,_|
                                                           
 v2.7.0
info: Website generation started at 2:45:43 PM
info: Building assets...
info: Assets built
info: Generating pages...
[=] 1 / 1 pages built
info: Pages built
info: Website generation complete! Total build time: 0.154s
info: Build success!
info: Site data built
Running test_site_templates/test_default test
  __  __                  _      ____    _               _ 
 |  \/  |   __ _   _ __  | | __ | __ )  (_)  _ __     __| |
 | |\/| |  / _` | | '__| | |/ / |  _ \  | | | '_ \   / _` |
 | |  | | | (_| | | |    |   <  | |_) | | | | | | | | (_| |
 |_|  |_|  \__,_| |_|    |_|\_\ |____/  |_| |_| |_|  \__,_|
                                                           
 v2.7.0
info: Initialization success.
  __  __                  _      ____    _               _ 
 |  \/  |   __ _   _ __  | | __ | __ )  (_)  _ __     __| |
 | |\/| |  / _` | | '__| | |/ / |  _ \  | | | '_ \   / _` |
 | |  | | | (_| | | |    |   <  | |_) | | | | | | | | (_| |
 |_|  |_|  \__,_| |_|    |_|\_\ |____/  |_| |_| |_|  \__,_|
                                                           
 v2.7.0
info: Website generation started at 2:45:45 PM
info: Building assets...
info: Assets built
info: Generating pages...
[=====] 5 / 5 pages built
info: Pages built
info: Website generation complete! Total build time: 0.408s
info: Build success!
info: Site data built
Test passed
```

**Comment by yamgent** [comment]
Hi, thanks for your report. Is Graphviz installed on your computer as well? If you create an empty MarkBind website and populate with the following content, does it work?

`index.md`:
```html
<puml src="abc.puml" width=300/>
```

`abc.puml`:
```puml
@startuml
alice -> bob ++ : hello
bob -> bob ++ : self call
bob -> bib ++  #005500 : hello
bob -> george ** : create
return done
return rc
bob -> george !! : delete
return success
@enduml
```

**Comment by le0tan** [comment]
I found the problem: I installed graphviz using scoop, and it didn't add it to the PATH variable list. It seems that installing graphviz using the installer on its official website won't add itself to PATH as well. After manually configuring PATH the site builds successfully.

Thanks for your in time response! Is it OK for me to submit a PR to include installing Java and Graphviz in the developer guide?
